### PR TITLE
[FIX-#4592]Fix that the worker group management IP is not displayed correctly.

### LINKED ISSUE #4592: [Improvement][api] Worker group data returned incorrectly
image
image

**Which version of DolphinScheduler:**
 -[dev]

### PR DESCRIPTION
**Pull request by zt-1997**
this close #4592

### CODE REVIEW

**Comment by zt-1997** [comment]
fix #https://github.com/apache/incubator-dolphinscheduler/issues/4592

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/WorkerGroupService.java [hunk 1]
```diff
@@ -159,13 +159,19 @@
...
             String workerGroupPath = workerPath + "/" + workerGroup;
             List<String> childrenNodes = zookeeperCachedOperator.getChildrenKeys(workerGroupPath);
+            String timeStamp = "";
+            for (int i = 0; i < childrenNodes.size(); i++) {
+                String ip = childrenNodes.get(i);
+                childrenNodes.set(i, ip.substring(0, ip.lastIndexOf(":")));
+                timeStamp = ip.substring(ip.lastIndexOf(":"));
+            }
             if (CollectionUtils.isNotEmpty(childrenNodes)) {
                 availableWorkerGroupList.add(workerGroup);
```
Comment (CalvinKirs) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/WorkerGroupService.java:169]: delete it

**Review by CalvinKirs — CHANGES_REQUESTED** [review]

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/WorkerGroupService.java (hunk 1 above)
Comment (zt-1997) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/WorkerGroupService.java:169]: > delete it
I don't know what you mean.
Delete which？

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
Very good, looking forward to your next contribution.

**Pull request merged**